lib/paperclip/attachment.rb

```diff
@@ -2,6 +2,7 @@
 require 'uri'
 require 'paperclip/url_generator'
 require 'active_support/deprecation'
+require 'active_support/core_ext/string/inflections'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.